specification/automation/resource-manager/Microsoft.Automation/stable/2022-01-31/deletedAutomationAccount.json

```diff
@@ -0,0 +1,147 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "AutomationManagement",
+    "version": "2022-01-31",
+    "x-ms-code-generation-settings": {
+      "useDateTimeOffset": true
+    }
+  },
+  "host": "management.azure.com",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [
+    "application/json"
+  ],
+  "produces": [
+    "application/json"
+  ],
+  "security": [
+    {
+      "azure_auth": [
+        "user_impersonation"
+      ]
+    }
+  ],
+  "securityDefinitions": {
+    "azure_auth": {
+      "type": "oauth2",
+      "authorizationUrl": "https://login.microsoftonline.com/common/oauth2/authorize",
+      "flow": "implicit",
+      "description": "Azure Active Directory OAuth2 Flow",
+      "scopes": {
+        "user_impersonation": "impersonate your user account"
+      }
+    }
+  },
+  "paths": {
+    "/subscriptions/{subscriptionId}/providers/Microsoft.Automation/deletedAutomationAccounts/{automationAccountName}": {
+      "get": {
+        "tags": [
+          "DeletedAutomationAccounts"
+        ],
+        "operationId": "deletedAutomationAccounts_ListBySubscription",
```

Comment:
>deletedAutomationAccounts_ListBySubscription [](http://example.com/codeflow?start=24&length=44)

Can you explain the use case for why this api is needed ? I am assuming this is a proxy resource since it only has a get
Seems like you are designing this to get older deleted accounts. How does the customer use this ? 

Generally it is better to model the list resource at subscription level ..to get subscroption level resources you would do something like : 

/subscriptions/{subscriptionId}/providers/Microsoft.Automation/deletedAutomationAccounts

without automation account name.

Reply:
deletedAutomationAccounts is resource type.
/subscriptions/{subscriptionId}/providers/Microsoft.Automation/deletedAutomationAccounts API returns list of deleted automation accounts with in a subscription. Use case of this API is to restore deleted accounts.